protocol/messenger_communities.go

```diff
@@ -733,12 +746,34 @@ func (m *Messenger) RemoveUserFromCommunity(id types.HexBytes, pkString string)
 	return response, nil
 }
 
+// TODO
+func (m *Messenger) SendKeyExchangeMessage(communityId []byte, pubkeys []*ecdsa.PublicKey) error {
+	rawMessage := common.RawMessage{
+		SkipEncryption: false,
+		Recipients:     pubkeys,
+		MessageType:    protobuf.ApplicationMetadataMessage_GROUP_KEY_EXCHANGE_MESSAGE,
+	}
+	_, err := m.sender.SendCommunityMessage(context.Background(), communityId, rawMessage)
+
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
 func (m *Messenger) BanUserFromCommunity(request *requests.BanUserFromCommunity) (*MessengerResponse, error) {
 	community, err := m.communitiesManager.BanUserFromCommunity(request)
 	if err != nil {
 		return nil, err
 	}
 
+	// TODO generate new encryption key
+	//communityKeyId, err := m.encryptor.GenerateHashRatchetKey(community.ID())
+	err = m.SendKeyExchangeMessage(community.ID(), community.GetMemberPubkeys())
```

Comment:
I would hide this behind a feature flag if not ready (it can just be a const for now)

Reply:
`SendKeyExchangeMessage` uses an extra parameter now designating whether key should be forcefully renewed.